.github/workflows/check.yml

```
@@ -0,0 +1,15 @@
+on: [push, pull_request]
+name: CI Checks
+jobs:
+  check:
+    name: Check and Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: thumbv6m-none-eabi
```

For a slightly faster execution time, this avoids installing `rust-docs` for nothing. I'm not sure the `override` is needed, but adding it makes us sure that it's using the version we want.

```suggestion
      - uses: actions-rs/toolchain@v1
        with:
          toolchain: stable
          target: thumbv6m-none-eabi
          override: true
          profile: minimal
          components: rustfmt, clippy
```